### PR TITLE
[10.0] Add module product_expiry_simple

### PR DESCRIPTION
**Pull request by alexis-via**
See README for more details about the module.

### CODE REVIEW

File: product_expiry_simple/models/stock_production_lot.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+from odoo import models, fields, api
+
+
+class StockProductionLot(models.Model):
+    _inherit = 'stock.production.lot'
+    _rec_name = 'display_name'
+
+    expiry_date = fields.Date(string='Expiry Date')
+    display_name = fields.Char(
+        compute='compute_display_name_field',
```
Comment (simahawk) [inline product_expiry_simple/models/stock_production_lot.py:14]: Just a nitpicking request: change compute method name respecting [contrib rule](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#field) for such field: `Compute Field: the compute method pattern is _compute_<field_name>`

**Review by simahawk — CHANGES_REQUESTED** [review]
Just code review, no test.
LGTM but the small change for computing method ;)

**Review by simahawk — COMMENTED** [review]
@alexis-via BTW what's the goal of the module and what you achieve really installing it? I understood it from the code but the README should explain what you get, not just how is different from the official one, isn't it? :)

File: product_expiry_simple/models/stock_production_lot.py (hunk 1 above)
Comment (alexis-via) [reply product_expiry_simple/models/stock_production_lot.py:14]: I named it that way on purpose: if you call the method compute_display_name, you have a warning in the logs of odoo v10 (this is something specific to display_name).

**Comment by alexis-via** [comment]
@simahawk I improved the README as suggested

**Review by cdubuit — APPROVED** [review]

File: product_expiry_simple/models/stock_production_lot.py (hunk 1 above)
Comment (simahawk) [reply product_expiry_simple/models/stock_production_lot.py:14]: I see. Good to know :)

**Review by simahawk — APPROVED** [review]

**Pull request merged**